### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by techmoves**
in this repo i did this:

re arrange the the component 

- [x] Created a new directory called logic 

   

- [x]  Copy the files calculate.js and operate.js into  logic/ directory

    

- [x] Install [big.js](https://www.npmjs.com/package/big.js) using NPM

    

- [x] Import calculate.js in your Calculator.js component

    

- [x] Implement click event handlers using the math logic from calculate.js and operate.js in your React component
- [x] no the Calculator can do functionalities

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
+    "react/prop-types": "off",
```
Comment (amejid) [inline .eslintrc.json]: - [x] Kindly use the original linter configuration files as they help us in ensuring we write clean and more organized code. Please don't alter the linter configuration files in any way.

**Review by amejid — COMMENTED** [review]
Hi @techmoves, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has a linter configuration file that is tampered with. Please use the correct configuration files.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json (hunk 1 above)
Comment (techmoves) [reply .eslintrc.json]: resolved thanks

Comment (amejid) [reply .eslintrc.json]: No, it's not resolved. Please fix the linter errors instead of altering the configuration.

**Review by amejid — COMMENTED** [review]
Hi @techmoves, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that has a linter configuration file that is tampered with. Please use the correct configuration files.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/componets/calculator.js [hunk 2]
```diff
@@ -1,14 +1,74 @@
...
+        </div>
+        <div className="row">
+          <Button className="calc-btn" value="7" handleClick={eventHandler} />
+          <Button className="calc-btn" value="8" handleClick={eventHandler} />
+          <Button className="calc-btn" value="9" handleClick={eventHandler} />
+          <Button
+            className="calc-btn orange"
+            value="*"
+            handleClick={eventHandler}
+          />
```
Comment (emmiiorji) [inline src/componets/calculator.js]: Well done for implementing the events for all buttons. However, I am not able to perform multiplication. Kindly fix it.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @techmoves, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<div align='center'>

</div>

 ## Good Highlights✔️

- Click events are implement for all buttons
- Good handling of state with `useState`

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @techmoves, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Calculator operations working as expected :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**